src/libecalc/dto/types.py

```diff
@@ -112,22 +110,13 @@ class InterpolationType(str, Enum):
 
 class FuelType(EcalcBaseModel):
     """An installation/model/component may change fuel over time, due to setup changes,
-    production, and the cost may also change on different predictive models.
+    production.
```

Comment:
Not your fault, I guess, but this sentence in the doctoring does not really make sense to me...

Reply:
I agree, should we just remove it - or do you have a suggestion to improve the doctoring? (e.g. "data transfer object for FuelType"?)

Reply:
I don't have any good suggestions. Just remove it, I guess.